tests/test_searcher.py

```diff
@@ -85,63 +93,115 @@ def _run_perf_analytics(
             fen,
             max_depth,
             enable_null_move_pruning=enable_null_move_pruning,
+            enable_futility_pruning=enable_futility_pruning,
             enable_delta_pruning=enable_delta_pruning,
             enable_transposition_table=enable_transposition_table,
             enable_aspiration_windows=enable_aspiration_windows,
         )
         profiler.disable()
         stats = pstats.Stats(profiler)
 
-        stats.strip_dirs().sort_stats("tottime").print_stats(10)
+        import os
+        import sys
+
+        test_name = (
+            test_name.replace("[", "_")
+            .replace("]", "_")
+            .replace(" ", "_")
+            .replace("/", "_")
+        )
+        perf_test_folder = "perf/"
+
+        if not os.path.exists(perf_test_folder):
+            os.mkdir(perf_test_folder)
+
+        with open(
+            os.path.join(perf_test_folder, f"{test_name}.txt"),
+            "w",
+        ) as f:
+            sys.stdout = f
+            print(
+                "------------------------------------------------------------------------------------------------"
+            )
+            print(f"FEN: {fen}")
+            stats = pstats.Stats(profiler)
+            stats.strip_dirs().sort_stats("tottime").print_stats()
+            print(
+                "------------------------------------------------------------------------------------------------"
+            )
+
+        sys.stdout = sys.__stdout__
 
     @pytest.mark.slow
-    def test_perf_base(self, fen_string: str, max_depth: int) -> None:
-        """Performance test base"""
```

Comment:
Just curious - why remove all the doc strings here?

Reply:
They don't provide any value since the docstring is one to one to the function name. I kept it where it wasn't like this